src/txorphanage.cpp

```diff
@@ -15,14 +14,35 @@ static constexpr int64_t ORPHAN_TX_EXPIRE_TIME = 20 * 60;
 /** Minimum time between orphan transactions expire time checks in seconds */
 static constexpr int64_t ORPHAN_TX_EXPIRE_INTERVAL = 5 * 60;
 
+void TxOrphanage::SubtractOrphanBytes(unsigned int size, NodeId peer)
```

Comment:
Imo, it would be nicer to do the orphanage changes separately, including amended functional/unit/fuzz tests.

Reply:
> Imo, it would be nicer to do the orphanage changes separately, including amended functional/unit/fuzz tests.

Opened #28481 for orphanage changes